✨(marsha) manage service worker path in nginx

lunika opened the pull request:
## Purpose

The service work we are using must be served at the root of the application. But we are managing it as a static file and is saved we the other static file managed by django. We need to add a rule in the nginx conf to access it from the root.


## Proposal

- [x] manage service worker path in nginx

lunika committed:
✨(marsha) manage service worker path in nginx

The service worker we are using must be served at the root of the
application. But we are managing it as a static file and it is saved
with the other static file managed by django. We need to add a rule
in the nginx conf to access it from the root.

sampaccoud commented on CHANGELOG.md:
```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
```
in commit message: 
s/service work/service worker
s/and is saved we the other static file/and IT is saved WITH the other static fileS

lunika committed:
🐛(gitlint) fix gitmoji source file url

The source file url used to check if a correct gitmoji is used in the
commit title has changer. We have to update this url in the
gitlint/gitlint_emoji.py script
